### PR TITLE
add brisk docs

### PR DESCRIPTION
**Pull request by Noviny**
Probably don't merge this yet, but I got brisk running in tangerine (The only content it finds is an empty readme atm - going to dig into displaying more stuff in a bit)

Discovery from this: We are missing a babel plugin (installed in our monorepo but not in website) trying to fix that atm.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "docs": "brisk dev",
+    "docs:build": "brisk build"
+  },
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/atlassian/tangerine.git"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (treshugart) [inline package.json]: @Blasz as set MIT in https://github.com/atlassian/tangerine/pull/39/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R1. There'll be some conflicts, but we probably want MIT. No action other than resolving to that when the time comes.

Comment (treshugart) [reply package.json]: In fact, we'll probably just merge `dependencies` and `scripts`.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "bugs": {
+    "url": "https://github.com/atlassian/tangerine/issues"
+  },
+  "homepage": "https://github.com/atlassian/tangerine#readme",
+  "dependencies": {
```
Comment (treshugart) [inline package.json]: `devDependencies`?

**Comment by nadiam84** [comment]
> I'm happy for this to go in whenever it's ready!

+100

**Comment by Blasz** [comment]
Closing since this has been implemented in #75.

**Pull request closed without merging**